README.md

```diff
@@ -4,25 +4,66 @@ Command line tool to upload/manage project on the [duke-data-service](https://gi
 
 Runs on Python 2.7 or 3.5.
 
-###Install:
+# Install:
 ```
 pip install DukeDSClient
 ```
 
-### Additional Setup:
-To use DukeDSClient, you must supply an authentication token.
+# Additional Setup:
 This requires a [Duke NetID](https://oit.duke.edu/email-accounts/netid/).
-This token identifies you with the service, and can be generated by visiting the [Duke Data Service API Explorer](https://api.dataservice.duke.edu/apiexplorer).
-Click the Authenticate button.
-Click Duke Authentication Service.
-Enter your NetId credentials.
-Your authentication token will then be printed at the top of the screen.
+To use DukeDSClient, it requires an agent key and a user key to be added to a config file.
 
-Note that tokens obtained this way are only valid for 2 hours. 
-Work is underway to improve this process and obtain longer-lived tokens.
+## Agent Key
+Hopefully your server will already have the global configuration setup.
+If not you can temporarily use the Agent key you create below.
+
+## Steps To Create a User Key
```

Comment:
Images look great. Have you considered moving this first time setup into a separate file and linking it? There are quite a few steps and images. But the trade off is that the instructions could be missed